### ISSUE TITLE
Default FTRL hyperparameters are very conservative

### ISSUE THREAD

**Issue by ataymano** [VowpalWabbit/vowpal_wabbit#4603]
### Describe the bug

Default FTRL learning rate is tiny ([ftrl_alpha = 0.005, ftrl_beta = 0.1](https://github.com/VowpalWabbit/vowpal_wabbit/blob/607460741e07bd094aa2834147a5526322b8c673/vowpalwabbit/core/src/reductions/ftrl.cc#L434))
Is there a specific reason for that? 
If so, maybe it should be mentioned in documentation, if not maybe it is better to adjust them

### How to reproduce

Prepare trivial dataset f(x) = 5
```
import random
with open('data.txt', 'w') as f:
    for i in range(100000):
        f.write(f'5 |n x:{random.random()}\n')
```

```
vw -d data.txt --ftrl -P 10000
```
Output:
```
Enabling FTRL based optimization
Algorithm used: Proximal-FTRL
ftrl_alpha = 0.005
ftrl_beta = 0.1
using no cache
Reading datafile = data.txt
num sources = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
Enabled learners: ftrl-Proximal-FTRL, scorer-identity, count_label
Input label = SIMPLE
Output pred = SCALAR
average  since         example        example        current        current  current
loss     last          counter         weight          label        predict features
16.87992 16.87992        10000        10000.0         5.0000         1.1049        2
14.21893 11.55794        20000        20000.0         5.0000         1.4351        2
12.43043 8.853419        30000        30000.0         5.0000         2.8207        2
11.09544 7.090500        40000        40000.0         5.0000         1.8357        2
10.02974 5.766940        50000        50000.0         5.0000         3.0356        2
9.153390 4.771614        60000        60000.0         5.0000         2.6330        2
8.426059 4.062073        70000        70000.0         5.0000         3.1193        2
7.803270 3.443745        80000        80000.0         5.0000         3.3855        2
7.268036 2.986167        90000        90000.0         5.0000         3.5255        2
6.796785 2.555522       100000       100000.0         5.0000         2.9113        2

finished run
number of examples = 100000
weighted example sum = 100000.000000
weighted label sum = 500000.000000
average loss = 6.796785
best constant = 5.000000
best constant's loss = 0.000000
total feature number = 200000
```

```
vw -d data.txt --ftrl -P 10000 --ftrl_alpha 0.1
```
Output:
```
Enabling FTRL based optimization
Algorithm used: Proximal-FTRL
ftrl_alpha = 0.1
ftrl_beta = 0.1
using no cache
Reading datafile = data.txt
num sources = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
Enabled learners: ftrl-Proximal-FTRL, scorer-identity, count_label
Input label = SIMPLE
Output pred = SCALAR
average  since         example        example        current        current  current
loss     last          counter         weight          label        predict features
0.302124 0.302124        10000        10000.0         5.0000         5.0000        2
0.151559 0.000994        20000        20000.0         5.0000         5.0000        2
0.101115 0.000227        30000        30000.0         5.0000         5.0000        2
0.075854 0.000074        40000        40000.0         5.0000         5.0000        2
0.060691 0.000038        50000        50000.0         5.0000         5.0000        2
0.050579 0.000020        60000        60000.0         5.0000         5.0000        2
0.043355 0.000011        70000        70000.0         5.0000         5.0000        2
0.037937 0.000010        80000        80000.0         5.0000         5.0000        2
0.033722 0.000005        90000        90000.0         5.0000         5.0000        2
0.030351 0.000004       100000       100000.0         5.0000         5.0000        2

finished run
number of examples = 100000
weighted example sum = 100000.000000
weighted label sum = 500000.000000
average loss = 0.030351
best constant = 5.000000
best constant's loss = 0.000000
total feature number = 200000
```

### Version

9.8.0

### OS

any

### Language

any

**Comment by ataymano** [comment]
@bremen79 , can you please help here? Is there a specific reason for such conservative default ftrl hyperparameters and if no, what would be better values?

**Comment by bremen79** [comment]
I went back in the history of the file and it seems John set the default values ~8 years ago and nobody ever changed them. 
The paper (https://dl.acm.org/doi/pdf/10.1145/2487575.2488200) does not mention default values.
I don't think there is a particular reason for these values. I would set it more or less to the same value used in SGD.

**Comment by peterychang** [comment]
done
